src/main/java/net/dries007/tfc/objects/blocks/devices/BlockBlastFurnace.java

```diff
@@ -47,29 +47,35 @@
     static
     {
         Predicate<IBlockState> stoneMatcher = state -> state.getBlock() instanceof BlockFireBrick;
-        Predicate<IBlockState> ironSheetMatcher = state -> (state.getBlock() instanceof BlockMetalSheet)
-            && ((BlockMetalSheet) state.getBlock()).getMetal() == Metal.WROUGHT_IRON;
+        Predicate<IBlockState> sheetMatcher = state -> {
+            if (state.getBlock() instanceof BlockMetalSheet)
+            {
+                BlockMetalSheet block = (BlockMetalSheet) state.getBlock();
+                return block.getMetal().getTier().isAtLeast(Metal.Tier.TIER_III);
```

Comment:
One thing to consider: As is, this allows metals that are technically "unfinished products", i.e. high carbon steels, weak steels, pig iron, etc. This *may* be fixed, likely just by requiring this as to also be `isToolMetal()` (which would restrict the list in TFC to iron, steel, or colored steel). However I'm unsure of if this change should actually be made